apps/dialog/package.json

```
@@ -47,6 +47,7 @@
     "@opentelemetry/sdk-node": "0.208.0",
     "@opentelemetry/sdk-trace-node": "2.2.0",
     "@opentelemetry/semantic-conventions": "1.38.0",
+    "@phosphor-icons/react": "^2.1.10",
```

Please use fixed versions.
```suggestion
    "@phosphor-icons/react": "2.1.10",
```